_observing-your-data/trace/distributed-tracing.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: default
+title: Distrbuted tracing 
+parent: Trace Analytics
+nav_order: 65
+---
+
+# Distributed tracing
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/6750).    
+{: .warning}
+
+Distributed tracing is used to monitor and debug distributed systems. You can track the flow of requests through the system and identify performance bottlenecks and errors. A _trace_ is a complete end-to-end path of a request as it flows through a distributed system. It represents the journey of a specific operation as it traverses various components and services in a distributed architecture. In distributed tracing, a single trace contains a series of tagged time intervals called _spans_. Spans have a start and end time, and may include other metadata like logs or tags to help classify what happened. 
+
+Use distributed tracing for the following purposes:
+
+- **Optimize performance:** Identify and resolve bottlenecks, reducing latency in your applications.
+- **Troubleshoot errors:** Quickly pinpoint the source of errors or unexpected behavior in your distributed system.
+- **Allocate resources:** Optimize resource allocation by understanding usage patterns of different services.
+- **Visualize service dependencies:** Visualize dependencies between services, helping you to manage architectures. 
+
+## Distributed tracing pipeline
+
+OpenSearch provides a distributed tracing pipeline that can be used to ingest, process, and visualize tracing data with query and alerting functionality. [OpenTelemetry](https://opentelemetry.io/) is an open-source observability framework that provides a set of APIs, libraries, agents, and collectors for generating, capturing, and exporting telemetry data. The distributed tracing pipeline consists of the following components: 
+
+- **Instrumentation:** Instrumenting your application code with OpenTelemetry SDKs.
```

Comment:
Should this and the preceding list match in format (We use the -ing in this list but not in the preceding one).

Reply:
Yes, good catch. Change made to preceding list to use -ing.